plenum/test/consensus/order_service/test_ordering_process_preprepare.py

```diff
@@ -114,7 +107,7 @@ def request_propagates(reqs):
 
 def test_process_pre_prepare_with_ordered_request(orderer, pre_prepare):
     handler = Mock()
-    orderer_with_requests._bus.subscribe(ThrowSuspiciousNode, handler)
+    orderer._bus.subscribe(ThrowSuspiciousNode, handler)
```

Comment:
Don't we need to call `orderer_with_requests` so that it's essential that we must have requests here?

Reply:
We don't need to call ```orderer_with_requests``` because in this test we patch ```db_manager.stores```, ```_non_finalised_reqs``` and don't need in any requests in the ```orderer```.